codesystems/CodeSystem-England-ODSDateTime.xml

```diff
@@ -0,0 +1,37 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
+  <id value="England-ODSDateTime" />
+  <url value="https://digital.nhs.uk/services/organisation-data-service/CodeSystem/England-ODSDateTime" />
+  <version value="1.1.0" />
+  <name value="EnglandODSDateTime" />
+  <title value="England ODS Date Time" />
+  <status value="active" />
+  <date value="2023-12-07" />
+  <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <contact>
+    <name value="Organisation Data Service" />
+    <telecom>
+      <system value="email" />
+      <value value="exeter.helpdesk@nhs.net" />
+      <use value="work" />
+      <rank value="2" />

```

Comment:
Not sure about rank=2, is that not the rank of multiple ContactPoint, whereas, these are two separate ContactDetail, each with a ContactPoint?

Reply:
That's a good point, agree with you, I'll revert it here and elsewhere.

Reply:
Reverted now, marking as resolved